### ISSUE TITLE
Minor bug on assembly step

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Owen-haha/mocat#2]
```
I figured out a minor bug at assembly step :

MOCAT.pl -sf MOCAT.sample -a -r hg19

At some point MOCAT is running zcat commands but tried to create temporary 
files in wrong directory.

The reason is simple.

My directory was like that:

/home/user/Mocat_sample_4/sample_4

Where Mocat_sample_4 is the working directory and sample_4 is the directory 
containing fastq, which I give in MOCAT.sample file.

Because 'sample_4' appear 2 times in my full path, MOCAT was trying to write 
temporary files in :

/home/user/Mocat_sample_4/temp

I changed the path to /home/user/Mocat_s4/sample_4

And MOCAT is working and writting in :

/home/user/Mocat_s4/sample_4/temps


```

Original issue reported on code.google.com by `pingouin...@gmail.com` on 21 Nov 2012 at 1:49

**Comment by GoogleCodeExporter** [comment]
```
Thank you for noticing this bug. We will look into this and correct it for the 
next release.
```

Original comment by `Jens.Kultima` on 21 Nov 2012 at 1:53
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue has now been fixed in upcoming version 1.2.
```

Original comment by `Jens.Kultima` on 26 Nov 2012 at 1:08
- Changed state: **Fixed**
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**
